examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -33,6 +38,9 @@
 epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
                     proj=True, baseline=(None, 0), preload=True,
                     reject=dict(grad=4000e-13, eog=150e-6))
+# Pull 2**n points to speed up computation
+epochs = EpochsArray(epochs.get_data()[..., :1024], epochs.info,
+                     epochs.events, tmin, epochs.event_id)
```

Comment:
It seemed to me when I tried this before that there were some inconsistencies with the indexing using times (e.g., it would be off by 1 index in one direction or the other), so if I ever want my signal to be length 2**N I just index the data directly. I can use tmin/tmax if you think that's better.